### PR TITLE
Update to version 0.6

### PR DESCRIPTION
**Pull request by m-rossi**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Not sure why the bot has not picked up the new version, but here is my manual upgrade.

### CODE REVIEW

**Comment by m-rossi** [comment]
@conda-forge-admin, please rerender

**Comment by m-rossi** [comment]
I think the reason the bot did not pick up the version update is may related to the source distribution file name change?

**Comment by m-rossi** [comment]
@conda-forge/pint-pandas this is ready for review.

**Comment by m-rossi** [comment]
@conda-forge/pint-pandas pinging again and asking for merge.

**Review by znicholls — APPROVED** [review]
@hgrecco @dopplershift lgtm. Any comments? If no, I will merge Friday at earliest

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,14 +19,14 @@ build:
...
+    - setuptools_scm >=3.4.3
+    - setuptools >=61
   run:
-    - python >=3.7
-    - setuptools
-    - pint >=0.13
-    - pandas >=1.0.4
+    - python >=3.9
+    - pint >=0.21
+    - pandas >=1.5
```
Comment (dopplershift) [inline recipe/meta.yaml]: While this does match what's in `pyproject.toml`, according to the changelog, it's Pint 0.24 and Pandas 2.2:

https://github.com/hgrecco/pint-pandas/blob/899dd266d6457f672c3b53f44f71e1e91a74ef25/CHANGES#L10-L18

Comment (m-rossi) [reply recipe/meta.yaml]: Thanks @dopplershift, I missed that.

**Comment by m-rossi** [comment]
@znicholls @dopplershift can we merge now?

**Comment by znicholls** [comment]
Yep let's go for it

**Pull request merged**